### PR TITLE
Get vive working

### PR DESCRIPTION
**Pull request by TrevorDev**

### CODE REVIEW

**Comment by TrevorDev** [comment]
https://github.com/BabylonJS/Babylon.js/issues/3362
https://github.com/BabylonJS/Babylon.js/issues/3352

**Comment by deltakosh** [comment]
All tests done on WMR and vive?

File: src/Cameras/VR/babylon.webVRCamera.ts [hunk 1]
```diff
@@ -160,6 +164,49 @@ module BABYLON {
...
+                // Add standing matrix offset to get real offset from ground in room
+                this._standingMatrix.getTranslationToRef(this._workingVector);
+                return this._deviceRoomPosition.y+this._workingVector.y
+            }else{
+                return this._defaultHeight;
+            }
+        }
+
+        public useStandingMatrix = ()=>{
+            return new Promise<boolean>((res, rej)=>{
```
Comment (deltakosh) [inline src/Cameras/VR/babylon.webVRCamera.ts]: Will it compile on es5 target?

Comment (TrevorDev) [reply src/Cameras/VR/babylon.webVRCamera.ts]: Nope, replaced with cb

**Comment by TrevorDev** [comment]
Yep, sponza, hillvalley and mansion work.

**Pull request merged**